.github/workflows/build.yml

```diff
@@ -126,40 +151,41 @@ jobs:
         arch: [arm, arm64, 386, amd64]
     env:
       repo: ${{ github.event.repository.name }}
-      version: ${{ needs.product-metadata.outputs.vault-version }}
+      version: ${{ needs.set-product-version.outputs.vault-version }}
     steps:
       - uses: actions/checkout@v3
       - uses: hashicorp/actions-docker-build@v1
         with:
           version: ${{ env.version }}
           target: default
           arch: ${{ matrix.arch }}
-          zip_artifact_name: ${{ env.PKG_NAME }}_${{ needs.product-metadata.outputs.vault-version }}_linux_${{ matrix.arch }}.zip
+          zip_artifact_name: ${{ env.PKG_NAME }}_${{ needs.set-product-version.outputs.vault-version }}_linux_${{ matrix.arch }}.zip
```

Comment:
Why don't we use env.version here like we do in ent instead of using needs.set-product-version.outputs.vault-version?